### ISSUE TITLE
problem importing matplotlib.animation in Catalina

### ISSUE THREAD

**Issue by djpine** [matplotlib/matplotlib#15637]
### Bug report

**Bug summary**

I cannot import matplotlib.animation (see error message below).  Importing matplotlib.pyplot works fine and I can create plots with it after the import.

**Code for reproduction**

```python
import numpy as np
import matplotlib.pyplot as plt
import matplotlib.animation as anim
```

**Actual outcome**

```
(base) davidpine@eng-dp82-001 Desktop % python junk.py
Traceback (most recent call last):
  File "junk.py", line 3, in <module>
    import matplotlib.animation as anim
  File "/Users/davidpine/opt/anaconda3/lib/python3.7/site-packages/matplotlib/animation.py", line 737, in <module>
    class ImageMagickWriter(ImageMagickBase, MovieWriter):
  File "/Users/davidpine/opt/anaconda3/lib/python3.7/site-packages/matplotlib/animation.py", line 120, in wrapper
    if writerClass.isAvailable():
  File "/Users/davidpine/opt/anaconda3/lib/python3.7/site-packages/matplotlib/animation.py", line 730, in isAvailable
    return super().isAvailable()
  File "/Users/davidpine/opt/anaconda3/lib/python3.7/site-packages/matplotlib/animation.py", line 427, in isAvailable
    return shutil.which(cls.bin_path()) is not None
  File "/Users/davidpine/opt/anaconda3/lib/python3.7/site-packages/matplotlib/animation.py", line 724, in bin_path
    binpath = mpl._get_executable_info('magick').executable
  File "/Users/davidpine/opt/anaconda3/lib/python3.7/site-packages/matplotlib/__init__.py", line 384, in _get_executable_info
    return impl([path, "--version"], r"^Version: ImageMagick (\S*)")
  File "/Users/davidpine/opt/anaconda3/lib/python3.7/site-packages/matplotlib/__init__.py", line 324, in impl
    args, stderr=subprocess.STDOUT, universal_newlines=True)
  File "/Users/davidpine/opt/anaconda3/lib/python3.7/subprocess.py", line 395, in check_output
    **kwargs).stdout
  File "/Users/davidpine/opt/anaconda3/lib/python3.7/subprocess.py", line 472, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/Users/davidpine/opt/anaconda3/lib/python3.7/subprocess.py", line 775, in __init__
    restore_signals, start_new_session)
  File "/Users/davidpine/opt/anaconda3/lib/python3.7/subprocess.py", line 1522, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
OSError: [Errno 86] Bad CPU type in executable: 'convert'
#
#
```

**Expected outcome**

**Matplotlib version**

  * Operating system: macos Catalina 10.15.1
  * Matplotlib version: 3.1.1
  * Matplotlib backend (`print(matplotlib.get_backend())`): MacOSX, Qt5Agg
  * Python version: 3.7.4
  * Jupyter version (if applicable): 
  * Other libraries:

**Comment by ImportanceOfBeingErnest** [comment]
This seems to be a duplicate of https://github.com/matplotlib/matplotlib/issues/15399.
Can you verify?

**Comment by anntzer** [comment]
I'm going to assume that this can be closed as a dupe of #15399.  Feel free to request a reopen if not.
